packages/ember-routing/lib/services/router.js

```diff
@@ -6,8 +6,33 @@ import { Service, readOnly } from 'ember-runtime';
 import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
 
 /**
-   The Router service is the public API that provides component/view layer
-   access to the router.
+   The Router service provides components, and other services, access
+   to the router.
+   
+   It can be used to check the current route, initiate route transitions and more.
+   
+   For example, it can be used to transition to a dedicated route if the 
+   application goes offline. Note that this is a boiled down example, 
+   in your app you should remove the event listener on service teardown.
+   
+   ```js
+   // app/services/network-connection.js
+   
+   import Service, { inject } from '@ember/service';
```

Comment:
`import Service, { inject as service } from '@ember/service';`
Closer to guides style.